### ISSUE TITLE
Image Text adjusting issues

### ISSUE THREAD

**Issue by mohamedshuaau** [kus/php-image#20]
First of all, I love the work you have done. This is the alternative I think we all were asking for. 

I am having a bit of an issue. Here is the default code from the docs which I copied for testing.

```
$image->text('This is a big sentence', array(
		'fontSize' => 60, // Desired starting font size
		'x' => 300,
		'y' => 200,
		'width' => 200,
		'height' => 50,
		'alignHorizontal' => 'center',
		'alignVertical' => 'center',
		'debug' => true
	));
```

This code wraps my text around a box (if i am not wrong) which has a width of 200px and 50px height correct? And the font size is 60. But on my image final render (save()), the texts are extremely small because it is trying to fit it in the box?

So I tried the next example:

  ```
$image->textBox('Auto wrap and scale font size to multiline text box width and height bounds. Vestibulum venenatis risus scelerisque enim faucibus, ac pretium massa condimentum. Curabitur faucibus mi at convallis viverra. Integer nec finibus ligula, id hendrerit felis.', array(
		'width' => 150,
		'height' => 140,
		'fontSize' => 16, // Desired starting font size
		'x' => 50,
		'y' => 150
	));
```

This works as intended but even if I write 100 for font size, it still wont change. I am using another languages font and imge-intervention/GD Lib works fine with that language. What am I doing wrong here? I just want to create a text which has a box surrounding it (so the text does not flow outside) and if the text is longer than the set width, it will go on to the next line. Also being able to center the text and change the font size. That was the requirement I had but the examples above does not help me in my case. What is the right way/what am I missing here?

Basically How can I not make the font size auto scale with the bounding box? Instead of scaling the font, keep the font size as it is and keep on moving the text to the next line if the text reaches the bounding box.

**Comment by kus** [comment]
Are you able to provide me example PHP files so I can see the issue?

I've only ever tried with English.
